python/ray/serve/deployment.py

```diff
@@ -162,6 +165,21 @@ def __init__(
                 raise ValueError("route_prefix may not contain wildcards.")
         if not (ray_actor_options is None or isinstance(ray_actor_options, dict)):
             raise TypeError("ray_actor_options must be a dict.")
+        if placement_group_bundles is not None:
```

Comment:
Sorry another q :), (for my knowledge), what is going to happen if i set pg in the placement_group_bundles and also num_cpus in the ray_actor_options?
e.g. I set bundle [{"CPU": 1}] and num_cpu=2.

Reply:
The `ReplicaConfig` will [fail validation](https://github.com/ray-project/ray/pull/37830/files#diff-2f5ba63ace082b7b7cfeceb2baf9b2d9f805f7559ac81e1fa5878e7b33aa69b9R531-R541).

Reply:
Thx! I missed that.

Reply:
FYI there's a test case for it in `test_replica_placement_group.py`